src/Dom/Document.php

```diff
@@ -6,14 +6,31 @@
 use AmpProject\DevMode;
 use AmpProject\Dom\Document\Encoding;
 use AmpProject\Dom\Document\Option;
+use AmpProject\Dom\Middleware\AmpBindAttributes;
+use AmpProject\Dom\Middleware\AmpEmojiAttribute;
+use AmpProject\Dom\Middleware\CharacterEncoding;
+use AmpProject\Dom\Middleware\DeDuplicateTag;
+use AmpProject\Dom\Middleware\DoctypeNode;
+use AmpProject\Dom\Middleware\HeadProfileToLink;
+use AmpProject\Dom\Middleware\HttpEquivCharset;
+use AmpProject\Dom\Middleware\InsertMissingCharset;
+use AmpProject\Dom\Middleware\LoadHtmlWithLibXmlFlags;
+use AmpProject\Dom\Middleware\MoveInvalidNodesToBody;
+use AmpProject\Dom\Middleware\MustacheScriptTemplates;
+use AmpProject\Dom\Middleware\NormalizeDocumentStructure;
+use AmpProject\Dom\Middleware\NormalizeHtmlAttributes;
+use AmpProject\Dom\Middleware\NoscriptElements;
+use AmpProject\Dom\Middleware\RemoveHttpEquivCharset;
+use AmpProject\Dom\Middleware\SelfClosingTags;
+use AmpProject\Dom\Middleware\SvgSourceAttributeEncoding;
```

Comment:
Instead of having individual imports for each of these, you should only import `AmpProject\Dom\Middleware` here, and then use relative namespace down below:
```suggestion
use AmpProject\Dom\Middleware;
```